bee-network/bee-autopeering/src/discovery/manager.rs

```diff
@@ -681,9 +681,9 @@ fn handle_verification_response(verif_res: VerificationResponse, verif_reqval: R
 
     // Send the response notification.
     if let Some(tx) = verif_reqval.response_tx {
-        // Panic: we don't allow channel send errors.
-        tx.send(verif_res.to_protobuf().to_vec())
-            .expect("error sending response signal");
+        // Note: if sending fails, then the response arrived a tiny bit too late for the predefined timeout (race
+        // condition), so we just ignore the error in that case.
+        let _ = tx.send(verif_res.to_protobuf().to_vec());
```

Comment:
Maybe we should emit a `warn` in those three cases?

Reply:
Would `debug` also fine for you? `warn` would indicate that the node could do anything about it but it really can't.

Reply:
Yeah, that's right. `debug` is fine with me 😊